apps/meteor/tests/end-to-end/api/livechat/contacts.ts

```diff
@@ -211,6 +223,59 @@ describe('LIVECHAT - contacts', () => {
 				expect(res.body).to.have.property('error');
 				expect(res.body.error).to.be.equal('Invalid value for Custom Field 1 field');
 			});
+
+			it('should remove an old custom field from a contact on update if it is not registered in the workspace anymore', async () => {
```

Comment:
can u add a test removing multiple custom fields?

Reply:
I added a test to validate optional custom fields are being removed when requested